ecpy_hqc_legacy/instruments/drivers/driver_tools.py

```diff
@@ -107,7 +107,7 @@ def __set__(self, obj, value):
             super(instrument_property, self).__set__(obj, value)
 
 
-def secure_communication(max_iter=2):
+def secure_communication(max_iter=10):
```

Comment:
why this change ?

Reply:
I was having issues with an instrument, and trying more times to connect helped, but in the meantime I found a better solution (communicate through SOCKET rather than LAN). Anyway, changed back.